aikau/src/main/resources/alfresco/accessibility/AccessibilityMenu.js

```diff
@@ -51,16 +52,18 @@
  define(["dojo/_base/declare",
          "dijit/_WidgetBase", 
          "dijit/_TemplatedMixin",
+         "alfresco/core/Core",
+         "alfresco/core/TopicsMixin",
          "dojo/text!./templates/AccessibilityMenu.html",
          "dojo/_base/lang",
-         "alfresco/core/Core",
+         "dojo/_base/array",
          "dojo/dom",
          "dojo/dom-construct",
          "dojo/on",
          "dojo/dom-attr"], 
-         function(declare, _WidgetBase, _TemplatedMixin, template, lang, AlfCore, dom, domConstruct, on, domAttr) {
+         function(declare, _WidgetBase, _TemplatedMixin, AlfCore, TopicsMixin, template, lang, array, dom, domConstruct, on, domAttr) {
    
-   return declare([_WidgetBase, _TemplatedMixin, AlfCore], {
+   return declare([_WidgetBase, _TemplatedMixin, TopicsMixin, AlfCore], {
```

Comment:
Core extends TopicsMixin, as topics will always be needed when publishing/subscribing, which is one of the main purposes of Core. Because of the nature of Javascript inheritance, this should not add any noticeable overhead, and it makes all of the code cleaner, otherwise every class would always have to extend/mixin Core and TopicsMixin.